openedx/features/edly/context_processor.py

```diff
@@ -1,11 +1,12 @@
 from math import floor
 
+from django.conf import settings
 from django.core.cache import cache
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 
 CACHE_NAME = 'context_processor.dynamic_theming'
 EDLY_CACHE_NAME = 'context_processor.edly_app'
-DEFAULT_SERVICES_COOKIE_EXPIRY = 180
+DEFAULT_SERVICES_NOTIFICATIONS_COOKIE_EXPIRY = 180
```

Comment:
nit: please add the measuring unit, for example 
```
DEFAULT_SERVICES_NOTIFICATIONS_COOKIE_EXPIRY = 180 # value in nano seconds 
```

Reply:
You have missed this feedback

Reply:
Feedback Implemented